src/app/shared/components/template/services/template-nav.service.ts

```diff
@@ -39,9 +39,7 @@ export class TemplateNavService {
     if (popup_child && !parent && popup_parent !== name) {
       await this.handlePopupActionsFromOther(params, container);
     }
-    if (popup_child && popup_child === name) {
-      await this.handlePopupActionsFromChild(params, container);
-    }
```

Comment:
this is just code tidying, the function here isn't actually called/used (instead uses the above `handlePopupActionsFromOther`